Yoast/duplicate-post
# Tests: use PSR-4

## Context

* Improve dev experience 

## Summary

This PR can be summarized in the following changelog entry:

* Tests: use PSR-4

## Relevant technical choices:

### Tests: make sure all files are namespaced

... including the test bootstrap.

Includes ensuring that all calls to global functions and global constants are now fully qualified.

### Tests: use PSR-4 

* Ensure subdirectory names use "ProperCase".
* Ensure all test class file names match the name of the test class.
* Change the Composer `autoload-dev` directive to use PSR-4 autoloading.
* Update the `phpunit.xml.dist` file to match.
* Update the `.phpcs.xml.dist` file to match.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* _N/A_ If the tests pass, we're good.
